src/utils/types.ts

```diff
@@ -50,12 +50,19 @@ interface AppStatus {
   outcome: string
 }
 
+interface ElementAndResponse {
+  element: ElementState,
+  response: ResponseFull
+}
```

Comment:
This is not being used anymore, is it?

Reply:
And if it is -- that comma is unnecessary (VSCode should remove it anyway)

Reply:
Have removed, along with SectionPages and SectionElements, which are no longer used.

Reply:
Yep. Cool! Agreed